packages/experiments-realm/crm/account.gts

```diff
@@ -522,6 +514,80 @@ class FittedTemplate extends Component<typeof Account> {
   </template>
 }
 
+class AtomTemplate extends Component<typeof Account> {
```

Comment:
Whats with this code?? You are creating an atom template for account? Where is this used? 

At first sight, I dont think you need to add this

Reply:
Even if it was an atom template, you are using contact to display the atom format of an account. To me, I'd rather have the account name which shud be derived from the computed name of the account